pkg/util/util.go

```diff
@@ -37,6 +37,8 @@ const (
 	memoryDumpOverhead = 100 * 1024 * 1024
 
 	UnprivilegedContainerSELinuxLabel = "system_u:object_r:container_file_t:s0"
+
+	PollImmediately = true
```

Comment:
Extending usage of `util`, `common`, `tools` or similar general unclassified packages in a no-go in terms of code-quality.

You can create a `wait` package under the `apimachinery` package and place it there.
If you do that, you could have wrappers there and expose two different functions to the rest of the project.

Reply:
@RokibulHasan7 Please address this comment